packages/compiler/src/core/helpers/type-name-utils.ts

```diff
@@ -164,8 +166,9 @@ function isInTypeSpecNamespace(type: Type & { namespace?: Namespace }): boolean
 
 function getModelPropertyName(prop: ModelProperty, options: TypeNameOptions | undefined) {
   const modelName = prop.model ? getModelName(prop.model, options) : undefined;
+  const defaultName = options?.operationName ?? "(anonymous mode)";
 
-  return `${modelName ?? "(anonymous model)"}.${prop.name}`;
+  return `${modelName ?? defaultName}.${prop.name}`;
```

Comment:
Was this used for debugging?

Reply:
this sounds hacky to me to have this operationName that replace anonymous models, we have an item to make those display better(like typescript does) I don't think we should add that parameter just to make it look a tiny bit better

Reply:
I'm fine to switch it back but then they will all ready "Namespace.(anonymous model).param". But I guess the diagnostic appears on the operation so it should be decipherable enough.

Reply:
yeah I totally get that the current state is bad but this is just a patch to an publicly exported util that we use everywhere so I'd rather not have this solution for now. If you want to have a better message just for this rule I woiuld maybe call a new function instead just for that(defined in the versioning file) that adds the prefix you want.

Reply:
I just reverted it for now and left the "(anonymous model)" language in place.